### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by StealthHydra179**
btw this is supposed to be in any repo that ppl collaborate on and contributing is all caps.

### CODE REVIEW

**Comment by StealthHydra179** [comment]
(dont merge yet)

**Comment by nobody5050** [comment]
Sure, if you want to follow a template I have a good one I can give you

**Comment by StealthHydra179** [comment]
sure

**Comment by nobody5050** [comment]
[here](https://gist.github.com/PurpleBooth/b24679402957c63ec426)

**Comment by StealthHydra179** [comment]
meh idk i might make one from scratch
depends how im feeling in 2 hrs lmao

**Comment by nobody5050** [comment]
haha welp.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
-# How to Contribute
+# Contributing
+When contributing this reposity fork this reposity and make a pull request. If we deem your edits to be worth enough value, we will merge your pull request.
```
Comment (nobody5050) [inline CONTRIBUTING.md]:
```suggestion
When contributing this repository fork this repository and make a pull request. If we deem your edits to be worth enough value, we will merge your pull request.
```
Possible typo?

Comment (nobody5050) [reply CONTRIBUTING.md]: I’ll let you fix it if it’s a typo.

Comment (StealthHydra179) [reply CONTRIBUTING.md]: ye il fix it later

**Comment by StealthHydra179** [comment]
@nobody5050 i put the stuff about project maintainers (but im not one reeet) uh also merge pls

**Comment by nobody5050** [comment]
Done

**Pull request merged**